src/argilla/server/search_engine.py

```diff
@@ -255,6 +255,8 @@ def _field_mapping_for_question(self, question: Question):
             return {"type": "text", "index": False}
         elif settings.type in [QuestionType.label_selection, QuestionType.multi_label_selection]:
             return {"type": "keyword"}
+        elif settings.type == QuestionType.ranking:
+            return {"type": "nested"}
```

Comment:
not sure if this is the best mapping type that we can use @frascuchon for the Ranking Question.

Reply:
It makes sense in terms we can query records with a response having the option o in ranking 1 for example. Let's keep it as is.